release.sh

```diff
@@ -0,0 +1,21 @@
+echo "Starting Release for Environment '$ENV'"
+
+# ----------- STAGING & REVIEW APPS --------------
+if [ \"$ENV\" = 'dev' ]; then
+  echo "=========== Development Release =============="
+  echo "----------- Set up DB Schema -----------------"
+  npm run db:reset -- --force
+  echo "----------- Seed DB with sample users --------"
+  npm run db:seed
```

Comment:
This moves seeding of the database from the web dyno's start to the release phase, so in DEV we from now on only reset when we deploy and not at each restart anymore (I find that cleaner and makes it easier to test for multiple days)